src/Layout/docs/styles.st.css

```
@@ -1,17 +1,18 @@
-@import '../../colors';
-
-.example-container {
+:import {
+  -st-from: "../../Foundation/stylable/colors.st.css";
+  -st-named: D70;
+}
+.exampleContainer {
   min-width: 966px;
   max-width: 1254px;
   padding: 30px 48px;
-
-  background: $D70;
+  background: D70;
```

In order to use Stylable variables you should wrap them with "value"
Like so:

```suggestion
  background: value(D70);
```